docs/source/osds/scenarios.rst

```diff
@@ -1,12 +1,232 @@
+input of devices.
+
+To use this configuration, the ``devices`` option must be populated with the
+raw device paths that will be used to provision the OSDs.
+
+
+.. note:: Raw devices must be "clean", without a gpt partition table, or
+          logical volumes present. Otherwise the tooling will refuse
+          to use them.
+
+For example, for a node that has ``/dev/sda`` and ``/dev/sdb`` intended for
+Ceph usage, the configuration would be:
+
+
+.. code-block:: yaml
+
+   osd_scenario: lvm
+   devices:
+     - /dev/sda
+     - /dev/sdb
+
+In the above case, if both devices are spinning drives, 2 OSDs would be
+created, one on each device, with a collocated journal.o
+
+Other provisioning strategies are possible, by mixing spinning and solid state
+devices, for example:
+
+.. code-block:: yaml
+
+   osd_scenario: lvm
+   devices:
+     - /dev/sda
+     - /dev/sdb
+     - /dev/nvme0n1
+
+Similar to the initial example, this would end up producing 2 OSDs, but data
+would be placed on the slower spinning drives (``/dev/sda``, and ``/dev/sdb``)
+and journals would be placed on the faster solid state device ``/dev/nvme0n1``.
+
+Other (optional) supported settings:
+
+- ``crush_device_class``: Sets the crush class for all OSDs created with this
+  method (it is not possible to have a per-OSD crush class using the *simple*
+  configuration approach). Values *must be* a string, like
+  ``crush_device_class: "ssd"``
+
+
+Advanced configuration
+^^^^^^^^^^^^^^^^^^^^^^
+
+This configuration is useful when more granular control is wanted when setting
+up devices and how they should be arranged to provision an OSD. It requires an
+existing setup of volume groups and logical volumes (the tooling will **not**
+create these).
+
+To use this configuration, the ``lvm_volumes`` option must be populated with
+logical volumes, volume groups, and optionally partitions for journals, or
+``block.db`` and ``block.wal``.
+
+.. note:: This configuration uses ``ceph-volume lvm create`` to provision OSDs
+
+Supported ``lvm_volumes`` configuration settings:
+
+- ``data``: The logical volume name or full path to a raw device (an LV will be
+  created using 100% of the raw device)
+
+- ``data_vg``: The volume group name. If using a logical volume name for ``data``
+    it is **required**.
+
+- ``crush_device_class``: Device class name for the resulting OSD, allows to
+  individually set the class for each OSD, unlike the global
+  ``crush_device_class`` that sets them for all OSDs.
+
+.. note:: If you wish to change set the ``crush_device_class`` for the OSDs
+          when using ``devices`` you must set it using the global ``crush_device_class``
+          option as shown above. There is no way to define a specific crush device class
+          per OSD when using ``devices`` like there is for ``lvm_volumes``.
+
+
+``filestore`` objectstore only values:
+
+- ``journal``: The logical volume name or full path to a partition.
```

Comment:
I assume this means "full device path", and if so, say so

Reply:
I didn't say "full device path" because I usually read "device" as `/dev/sda`, while here we want to make sure it is understood we need a partition. Maybe I can reinforce the path section by mentioning *or absolute path* instead